packages/core/demo/data/index.ts

```diff
@@ -184,136 +181,129 @@ let allDemoGroups = [
 				chartType: chartTypes.GroupedBarChart,
-				isDemoExample: true,
 			},
 			{
 				options: barDemos.groupedHorizontalBarEmptyStateOptions,
 				data: barDemos.groupedHorizontalBarEmptyStateData,
 				chartType: chartTypes.GroupedBarChart,
-				isDemoExample: true,
+				isDemoExample: false,
 			},
 			{
 				options: barDemos.groupedHorizontalBarSkeletonOptions,
 				data: barDemos.groupedHorizontalBarSkeletonData,
 				chartType: chartTypes.GroupedBarChart,
-				isDemoExample: true,
+				isDemoExample: false,
 			},
 			{
 				options: barDemos.stackedHorizontalBarOptions,
 				data: barDemos.stackedHorizontalBarData,
 				chartType: chartTypes.StackedBarChart,
-				isDemoExample: true,
 			},
 			{
 				options: barDemos.stackedHorizontalBarTimeSeriesOptions,
 				data: barDemos.stackedHorizontalBarTimeSeriesData,
 				chartType: chartTypes.StackedBarChart,
-				isDemoExample: true,
 			},
 			{
 				options: barDemos.stackedHorizontalBarEmptyStateOptions,
 				data: barDemos.stackedHorizontalBarEmptyStateData,
 				chartType: chartTypes.StackedBarChart,
-				isDemoExample: true,
+				isDemoExample: false,
 			},
 			{
 				options: barDemos.stackedHorizontalBarSkeletonOptions,
 				data: barDemos.stackedHorizontalBarSkeletonData,
 				chartType: chartTypes.StackedBarChart,
-				isDemoExample: true,
+				isDemoExample: false,
 			},
 		],
 	},
 	{
 		title: "Bubble",
+		description: "Bubble charts use data points and bubbles to plot measures anywhere along a scale. One measure is plotted along each axis. The size of the bubble represents the third measure. You can use bubble charts to represent financial data or any data where measured values are related.",
 		demos: [
 			{
 				options: bubbleDemos.bubbleDoubleLinearOptions,
 				data: bubbleDemos.bubbleDoubleLinearData,
 				chartType: chartTypes.BubbleChart,
-				isDemoExample: true,
 			},
 			{
 				options: bubbleDemos.bubbleTimeSeriesOptions,
 				data: bubbleDemos.bubbleTimeSeriesData,
 				chartType: chartTypes.BubbleChart,
-				isDemoExample: true,
 			},
 			{
 				options: bubbleDemos.bubbleDiscreteOptions,
 				data: bubbleDemos.bubbleDiscreteData,
 				chartType: chartTypes.BubbleChart,
-				isDemoExample: true,
 			},
 			{
 				options: bubbleDemos.bubbleEmptyStateOptions,
 				data: bubbleDemos.bubbleEmptyStateData,
 				chartType: chartTypes.BubbleChart,
-				isDemoExample: true,
+				isDemoExample: false,
 			},
 			{
 				options: bubbleDemos.bubbleSkeletonOptions,
 				data: bubbleDemos.bubbleSkeletonData,
 				chartType: chartTypes.BubbleChart,
-				isDemoExample: true,
+				isDemoExample: false,
 			},
 		],
 	},
 	{
 		title: "Donut",
+		description: "Bubble charts use data points and bubbles to plot measures anywhere along a scale. One measure is plotted along each axis. The size of the bubble represents the third measure. You can use bubble charts to represent financial data or any data where measured values are related.",
```

Comment:
This doesn't seem to be the right description

Reply:
```suggestion
```